src/jackdaw/admin.clj

```diff
@@ -6,11 +6,54 @@
   Like the underlying `AdminClient` API, this namespace is subject to
   change and should be considered of alpha stability."
   {:license "BSD 3-Clause License <https://github.com/FundingCircle/jackdaw/blob/master/LICENSE>"}
-  (:require [jackdaw.data :as jd])
-  (:import [org.apache.kafka.clients.admin
-            AdminClient DescribeTopicsOptions]))
-
-(set! *warn-on-reflection* true)
+  (:require
+   [jackdaw.data :as jd]
+   [manifold.deferred :as d])
+  (:import [org.apache.kafka.clients.admin AdminClient
+            DescribeTopicsOptions DescribeClusterOptions DescribeConfigsOptions]))
```

Comment:
I don't think DescribeTopicsOptions is used.
```suggestion
            DescribeClusterOptions DescribeConfigsOptions]))
```

Reply:
I made the client-impl use it for consistency with the other similar API calls. It seems like the MockAdminClient is more likely to support the variant that supplies the options so explicitly supplying those has benefits from a testing point of view.

Reply:
Ah apologies didn't see! Ill resolve.